src/Caching/Test/ClassesTests.cs

```diff
@@ -150,5 +150,21 @@ class B(A):
             // In JSON, class A should have 'class A doc' documentation while B should have none.
             Baseline.CompareToFile(BaselineFileName, json);
         }
+
+         [TestMethod, Priority(0)]
+        public void ClassesWithSuper() {
```

Comment:
I would recommend writing the test (almost) as though it worked, then sticking `[Ignore("some message")]` on it, rather than something commented out.

Reply:
updated